### PR TITLE
#1741 Update Publishing Workflow

### PR DESCRIPTION
**Pull request by Paul-Kawalkowski**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by matthoward366 — APPROVED** [review]
Looks good.

File: scripts/publish_modules.sh [hunk 1]
```diff
@@ -19,43 +19,13 @@
...
-		git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-		git fetch
-}
-
-setup_git_branch
-checkout_branch ${MAIN}
 
 cd ./canvas_modules/common-canvas
-npm version minor
+release=$(echo $GITHUB_REF | cut -d'v' -f2)
```
Comment (matthoward366) [inline scripts/publish_modules.sh:24]: This works but would GITHUB_REF_NAME be better?  Probably doesn't matter though.

Comment (Paul-Kawalkowski) [reply scripts/publish_modules.sh]: Changed it, prob better just in case we run into some weird case

**Review by matthoward366 — APPROVED** [review]

**Comment by matthoward366** [comment]
@Paul-Kawalkowski Can you create an issue and link it?  This would be under https://github.com/elyra-ai/canvas/issues.  Can be generic.

**Comment by Paul-Kawalkowski** [comment]
https://github.com/elyra-ai/canvas/issues/1741

**Pull request merged**